app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,6 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+    @actors = @movie.actors
```

Comment:
This isn't really necessary since we can also access `.actors` from the `@movie` instance that we send to the view.